history/4499.md

```diff
@@ -0,0 +1 @@
+* BMurri: WIXBUG:4499 - Bind MBApreq language to correct langid
```

Comment:
The current approach is old, but not *incorrect*. You need different naming and descriptions, like `UseUILanguages` or such.